### PR TITLE
.travis.yml: Enhance travis to upload wheel

### LINKED ISSUE #3459: Enhance Travis to upload wheel to testpypi
This is the first step in #3432

### PR DESCRIPTION
**Pull request by Shade5**
Travis uploads wheel to testpypi

Closes #3459

### Checklist

- [x] I have rebased properly. Please see [our tutorial on rebasing](http://coala.io/git#rebasing).
- [x] I have gone through the [commit guidelines](http://coala.io/commit) and I've followed them.
- [x] I have followed the [guidelines for the commit shortlog](http://coala.io/commit#shortlog).
- [x] I have followed the [guidelines for the commit body](http://coala.io/commit#commit-body).
- [x] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://coala.io/commit#issue-reference).
- [x] I have [run all the tests](http://api.coala.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### CODE REVIEW

**Comment by Shade5** [comment]
Should I create the `.pypirc` file too?

File: .travis.yml [hunk 1]
```diff
@@ -73,7 +73,15 @@ script:
...
     fi
+    source ./rultor_secrets.sh
   - bash .misc/deploy.coverage.sh
 
+deploy:
+  provider: pypi
+  server: https://testpypi.python.org/pypi
+  user: "$PYPIUSER"
+  password: "$PYPIPW"
+  distributions: "bdist_wheel" # Your distributions here
```
Comment (Mixih) [inline .travis.yml]: Hold up there, I don't think we should be deploying from travis without discussion. Right now we use rultor which provides an easy way to provide secrets. We might actually want you to modify rultor instead.

Comment (Shade5) [inline .travis.yml]: Sorry, I'll wait then

**Comment by jayvdb** [comment]
Continued at https://github.com/coala/coala/issues/3491

**Pull request closed without merging**